spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -292,7 +292,7 @@ def step(  # noqa
                 if success:
                     # Just leave the object on the receptacle if desired
                     if self.config.DONT_PICK_UP:
-                        print("open_gripper in don't pick up")
+                        self.say("open_gripper in don't pick up")
```

Comment:
There is no need for this change. Let's keep it `print`. No need to emit this as audio-out.